app/services/trainees/create_from_dttp.rb

```diff
@@ -136,9 +136,9 @@ def trn
     end
 
     def trainee_gender
-      # TODO: Need to take a decision on mapping gender other/not provided
-      # Hash#invert might not be desirable as we lose one of the duplicated values
-      Dttp::Params::Contact::GENDER_CODES.invert[dttp_trainee.response["gendercode"].to_i]
+      return :other if Dttp::Params::Contact::OTHER_GENDER_CODE == dttp_trainee.gender_code
+
```

Comment:
Just a thought - since we have an option for it, should we map to `gender_not_provided` if `dttp_trainee.gender_code` is nil?

I haven't looked at the numbers though - maybe it's always provided in DTTP 🤷‍♀️

Reply:
great idea, just checked, and it seems like we have a few records (~72), but that should definitely work.